app/frontend/components/common/asset.tsx

```diff
@@ -46,11 +61,13 @@ export const FormattedAssetItem = ({
     formattedUrl: h.JSX.Element | null
   }) => h.JSX.Element
 }) => {
+  const missingValue = 'N / A'
+
   const metadata = useSelector((state) =>
     state.tokensMetadata.get(createTokenRegistrySubject(policyId, assetName))
   )
 
-  const Icon = () => {
+  const icon = (() => {
```

Comment:
this is still a JSX element soI don't see a reason to rename it and not use it as `<Icon/>` in the code, or am I missing something? I'd also expect this element to receive the metadata as a parameter rather than using the `metadata` variable declared in the upper level scope, which seems pretty dirty even in the original code

Reply:
Refactored